Exception/InvalidParameterException.php

```diff
@@ -44,7 +44,7 @@ public static function withViolations(ParamInterface $parameter, ConstraintViola
             $message .= sprintf(
                 'Parameter "%s" of value "%s" violated a constraint "%s"',
                 $parameter->getName(),
-                is_scalar($invalidValue) ? $invalidValue : var_export($invalidValue, true),
+                is_scalar($invalidValue) ? utf8_encode($invalidValue) : var_export($invalidValue, true),
```

Comment:
`utf8_encode` requires the xml writer extension. it should be available most of the time so I guess this is fine.
But this will just change the original value so the error message becomes useless because the allegedly wrong value is not the real wrong value anymore. What I would propose, is to check if the string is valid utf8 and if not, hex encode the binary data and output the hex values of the wrong value.